app/views/api/advocates/claims/edit.jbuilder

```diff
@@ -0,0 +1 @@
+json.(@claim, :attributes)
```

Comment:
is there an argument we don't need jbuilder right now and can just return @claim.json ?